core/lib/snapshots_applier/src/lib.rs

```diff
@@ -109,16 +74,27 @@ impl From<RpcError> for SnapshotsApplierError {
     }
 }
 
+/// Main node API used by the [`SnapshotsApplier`].
```

Comment:
Nit: It's probably not worth to remove this comment, but does it add value? Maybe I've spent too much time in this part of the system, but for me the same set of info is encoded in the name of the type